components/badge/badge.html

```diff
@@ -3,66 +3,221 @@
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <link rel="stylesheet" href="./badge.css" />
+    <link rel="stylesheet" href="../../styles.css" />
     <link rel="stylesheet" href="../../core.css" />
+    <link rel="stylesheet" href="../../utils/prism/prism.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Pixel Design / Badge</title>
+    <title>Pixel Design / Documentation</title>
   </head>
   <body>
-    <div class="comp_wrapper">
-      <h2 class="comp_head-text">avatar</h2>
-      <p class="comp_desc-text p-lg">
-        The avatar component has 5 sizes,
-        <span class="comp_desc_highlight"> avatar-xl</span>,
-        <span class="comp_desc_highlight"> avatar-lg</span>,
-        <span class="comp_desc_highlight">avatar-md</span>,
-        <span class="comp_desc_highlight">avatar-sm</span>,
-        <span class="comp_desc_highlight">avatar-xs</span>
-      </p>
-      <h3 class="comp_head-text">standard avatars</h3>
-      <div class="wrapper_comp_box p-lg">
-        <div class="wrapper_av flex">
-          <span class="avatar av-xl">
-            <img src="../../assets/av.png" alt="avatar" class="rd" />
-            <div
-              title="Square online badge"
-              class="av_badge_lg online rd"
-            ></div>
-          </span>
-          <span class="avatar av-lg">
-            <img src="../../assets/av.png" alt="avatar" class="sq" />
-            <div
-              title="Square offline badge"
-              class="av_badge_md offline sq"
-            ></div>
-          </span>
-          <span class="avatar av-md">
-            <img src="../../assets/av.png" alt="avatar" class="rd" />
-            <div title="Square Idle badge" class="av_badge_sm idle rd"></div>
-          </span>
-          <span class="avatar av-sm">
-            <img src="../../assets/av.png" alt="avatar" />
-          </span>
-          <span class="avatar av-xs">
-            <img src="../../assets/av.png" alt="avatar" />
-          </span>
+    <!-- Documentation wrapper  -->
+    <div class="container dark_theme">
+      <!-- Navbar  -->
+      <nav class="wrapper_nav flex">
+        <h1>
+          <a href="../../index.html">
+            <span title="pixel-ui" class="h2 nav_brand_text"> PIXEL</span></a
+          >
+          <!-- <span>
+            <img src="./assets/splash.svg" alt="logo" class="nav_brand" />
+          </span> -->
+        </h1>
+
+        <div class="wrapper_nav_menu">
+          <ul class="nav_menu flex">
+            <li class="p">
+              <a
+                href="https://twitter.com/gauravsinhaweb"
+                rel="noopener noreferrer"
+                target="_blank"
```

Comment:
Setting target="_blank" on <a> elements now implicitly provides the same rel behavior as setting rel="noopener"
Read:
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Reply:
> Setting target="_blank" on elements now implicitly provides the same rel behavior as setting rel="noopener" Read: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

thanks for sharing this, I will improve it.